### ISSUE TITLE
Specify backend parameters with prefixes in projects.cfg

### ISSUE THREAD

**Issue by osma** [NatLibFi/Annif#126]
Current situation:

```
[fasttext-fi]
name=fastText Finnish
language=fi
backends=fasttext
analyzer=snowball(finnish)
dim=500
lr=0.25
epoch=30
loss=hs
limit=100
chunksize=24
```

Desired situation:

```
[fasttext-fi]
name=fastText Finnish
language=fi
backends=fasttext
analyzer=snowball(finnish)
fasttext.dim=500
fasttext.lr=0.25
fasttext.epoch=30
fasttext.loss=hs
fasttext.limit=100
fasttext.chunksize=24
```

By using the backend name as a prefix (similar to how `--backend-param` works on the CLI), we can support (with backend aliasing, which will become a separate issue) multiple backends of the same type for the same project. Also it makes it clear whether parameters are project- or backend-specific.

**Comment by osma** [comment]
This may become unnecessary after #218

**Comment by osma** [comment]
Not necessary anymore due to #218.
